core/controller/cloudevent/controller.go

```diff
@@ -61,6 +61,9 @@ func (c *controller) CloudEvent(ctx context.Context, wpr *WrappedPipelineRun) (e
 	if err != nil {
 		return err
 	}
+	// TODO(zhuxu): logs for troubleshooting of missing buildLog
```

Comment:
Why this is TODO

Reply:
mark it for deletion after the problem is resloved